data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```diff
@@ -131,6 +167,27 @@
     "copy-from": "milk_choc",
     "description": "Baby chocolate cow food, appropriated for adult humans.  Spoils rapidly."
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "salted_chocolate",
+    "name": { "str_sp": "salted chocolate" },
+    "weight": "258 g",
+    "color": "white",
+    "spoils_in": "1 d",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": -10,
+    "healthy": -15,
+    "calories": 132,
+    "description": "Salted liquid chocolate.  Spoils rapidly.  Tastes awful.",
+    "price": 38,
+    "price_postapoc": 5,
+    "material": [ "milk" ],
```

Comment:
@Fris0uman do you want me to create a cacao material or leave this as made of milk? Or could I change it to vegetable matter? Can't remember if that's a material.

Reply:
Having it use its own material would be nice, also I don't think cacao actually spoils rapidly

Reply:
I've changed it to Junk food and changed the description.  I think I've hit as far as I can go with this right now.

Reply:
Alright sounds good to me. We can always revisit it later if we really want cacao as a material.